inlong-common/src/main/java/org/apache/inlong/commons/metrics/MetricObserver.java

```diff
@@ -43,35 +47,36 @@
 
     /**
      * init
-     * 
+     *
      * @param commonProperties
      */
     public static void init(Map<String, String> commonProperties) {
         if (!isInited.compareAndSet(false, true)) {
             return;
         }
-        // init
-        Context context = new Context(commonProperties);
         // get domain name list
-        String metricDomains = context.getString(MetricListener.KEY_METRIC_DOMAINS);
+        String metricDomains = commonProperties.get(MetricListener.KEY_METRIC_DOMAINS);
         if (StringUtils.isBlank(metricDomains)) {
             return;
         }
         // split domain name
         String[] domains = metricDomains.split("\\s+");
         for (String domain : domains) {
             // get domain parameters
-            Context domainContext = new Context(
-                    context.getSubProperties(MetricListener.KEY_METRIC_DOMAINS + "." + domain + "."));
-            List<MetricListener> listenerList = parseDomain(domain, domainContext);
+            Map<String, String> domainMap = getSubProperties(commonProperties,
+                    MetricListener.KEY_METRIC_DOMAINS + "." + domain + ".");
+            List<MetricListener> listenerList = parseDomain(domainMap);
             // no listener
             if (listenerList == null || listenerList.size() <= 0) {
                 continue;
             }
             // get snapshot interval
-            long snapshotInterval = domainContext.getLong(MetricListener.KEY_SNAPSHOT_INTERVAL, 60000L);
-            LOG.info("begin to register domain:{},MetricListeners:{},snapshotInterval:{}", domain, listenerList,
-                    snapshotInterval);
+            long snapshotInterval =
+                    Long.parseLong(domainMap.getOrDefault(MetricListener.KEY_SNAPSHOT_INTERVAL,
```

Comment:
It is better that domainMap type is Context.
It can be removed that "60000" parse to long.

Reply:
If inlong-common do not want to import flume dependency, same function can be implemented in this class.

Reply:
> If inlong-common do not want to import flume dependency, same function can be implemented in this class.

Context need flume dependency, so Context is not used here.